@stellar/lyra-api/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@stellar/lyra-api",
+  "main": "build/index.min.js",
+  "types": "build/@stellar/lyra-sdk/src/index.d.ts",
```

Comment:
ick - i gotta fix this. wrestling a little with how tsconfig generates type declarations